src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,17 @@
+import classNames from 'classnames';
+
+export const Loader = () => {
+  return (
+    <div
+      data-cy="TodoLoader"
+      className={classNames(
+        'modal',
+        'overlay',
+        'is-active',
```

Comment:
you need a condition for `is-active` otherwise it would be active for every todo

Reply:
I'm using Loader component already with condition in another files